contracts/Core/JobManager.sol

```diff
@@ -50,7 +48,7 @@ contract JobManager is ACL, JobStorage {
     ) external payable 
     {
         numJobs = numJobs + 1;
-        uint256 epoch = stateManager.getEpoch();
+        uint256 epoch = constants.getEpoch();
```

Comment:
see here with ```constants.getEpoch();``` I get intuition that epoch is some constant that you are returning.
While with ```stateManager.getEpoch()``` there is no such confusion. 
As the state is an entity **I think we should have StateManager like earlier**

Reply:
I don't think its justifiable to spend more gas (few thousands) & redundant external calls (which can be trimmed) here.

If we are talking about intuition, then I think `Constants`  name is also wrong since contract will have values that can be modified by admin at some point in future.

Need to think of a suitable name, Its clear `Constants` doesn't work :)

Reply:
Yep

Reply:
maybe Governance.sol ?

Reply:
> maybe Governance.sol ?

Hmm, we don't have governance logic/mechanism in the contract yet, so it would be misleading IMO. WDYT @hrishikeshio @SkandaBhat @SamAg19 ?

Reply:
Agreed ^ we can have parameters.sol maybe.

Reply:
tbh, Parameters.sol sounds good